tests/backup/backup_kubevirt_test.go

```diff
@@ -2105,3 +2105,468 @@ var _ = Describe("{KubevirtVMRestoreWithAfterChangingVMConfig}", Label(TestCaseL
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+// This testcase verifies default backup & restore with both Kubevirt VMs and non-kubevirt resources
+var _ = Describe("{DefaultBackupRestoreWithKubevirtAndNonKubevirtNS}", func() {
```

Comment:
Please do not forget to make an entry to test case labels.

Cc: @mkoppal-px

Reply:
Yes. Please refer https://github.com/portworx/torpedo/pull/2225